.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,54 @@
+name: Bug report 🐞
+description: File a bug report
+title: "🐞[Bug]: "
+labels: 'bug'
+body:
+  - type: checkboxes
+    id: existing-issue
+    attributes:
+      label: Is there an existing issue for this?
+      description: Please search to see if an issue already exists for the bug you encountered.
+      options:
+        - label: I have searched the existing issues
```

Comment:
should we add something like "No, I have searched ..."?

Reply:
@growupanand  I think there shouldn't be an option here. Here, it is asking the issue raiser that if he checked the other issue similar to his before raising it otherwise it will be a duplicate issue.

Reply:
Yes, I mean same , just to add "No" word in starting to make message clear.